src/mlpack/methods/ann/rnn_impl.hpp

```diff
@@ -201,14 +201,19 @@ double RNN<OutputLayerType, InitializationRuleType, CustomLayers...>::Evaluate(
   ResetCells();
 
   double performance = 0;
+  size_t respSeq = 0;
```

Comment:
Do you think it would be useful to add a simple check on the shape of the `responses` if `single` is true or false?  Basically if `single == true`, we should ensure that the responses only have one slice.  And I think we should also be sure to point this out in the documentation.

Reply:
I need to go through the code for the output to do this. Let me check.